examples/sqlite/test/tests.ts

```diff
@@ -2,28 +2,525 @@ import * as dns from 'node:dns';
+                        responseJson.user.username === 'lastmjs_updated' &&
+                        responseJson.user.age === 35
+                };
+            }
+        },
+        {
+            name: '/posts patch 1',
+            test: async () => {
+                const response = await fetch(`${origin}/posts`, {
+                    method: 'PUT',
+                    headers: [['Content-Type', 'application/json']],
+                    body: JSON.stringify({
+                        id: 1,
+                        title: 'Post 1 title has changed again'
+                    })
+                });
+                const responseJson = await response.json();
+
+                return {
+                    Ok:
+                        responseJson.id === 1 &&
+                        responseJson.title ===
+                            'Post 1 title has changed again' &&
+                        responseJson.body === 'Post 1 body has changed' &&
+                        responseJson.user.id === 1 &&
+                        responseJson.user.username === 'lastmjs_updated' &&
+                        responseJson.user.age === 35
+                };
+            }
+        },
+        {
+            name: '/posts not empty',
+            test: async () => {
+                const response = await fetch(`${origin}/posts`);
+                const responseJson = await response.json();
+
+                return {
+                    Ok: responseJson.length === 1
+                };
+            }
+        },
+        {
+            name: '/posts/count 1',
+            test: async () => {
+                const response = await fetch(`${origin}/posts/count`);
+                const responseJson = await response.json();
+
+                return {
+                    Ok: responseJson === 1
+                };
+            }
+        },
+        {
+            name: '/posts/1 not null',
+            test: async () => {
+                const response = await fetch(`${origin}/posts/1`);
+                const responseJson = await response.json();
+
+                return {
+                    Ok:
+                        responseJson.id === 1 &&
+                        responseJson.title ===
+                            'Post 1 title has changed again' &&
+                        responseJson.body === 'Post 1 body has changed' &&
+                        responseJson.user.id === 1 &&
+                        responseJson.user.username === 'lastmjs_updated' &&
+                        responseJson.user.age === 35
+                };
+            }
+        }
+    ];
+}
+
+function usersTestsAfterBatch(origin: string): Test[] {
+    return [
+        {
+            name: '/users not empty',
+            test: async () => {
+                const response = await fetch(`${origin}/users`);
+                const responseJson = await response.json();
+
+                return {
+                    Ok: responseJson.length === 800
```

Comment:
I'm confused, shouldn't this be 500?

Reply:
Also should we have a constant instead of magic numbering this everywhere?

Reply:
Why should it be 500?

Reply:
It should be 800 because every post must create a user. I've added constants to make this more clear